docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -346,6 +346,9 @@ If you only need create or update commands, use the [az deployment group create]
 
 If generic update parameters and [az resource](../latest/docs-ref-autogen/resource.yml) don't meet your needs, you can use the [az rest](/cli/azure/reference-index#az-rest) command to call the REST API. The command automatically authenticates using the logged-in credential and sets header `Content-Type: application/json`. For more information, see [Azure REST API reference](/rest/api/azure/).
 
+> [!NOTE]
+> Calling a service REST API directly with `az rest` is a _supplemental_ feature of Azure CLI, and only works if the REST API is invoked correctly. Calling these APIs with `az rest` is subject to change for reasons including, but not limited to, Azure CLI permission changes, Microsoft Graph service limitations, AAD Conditional Access policies, etc. For service REST API questions or issues, please [create a support ticket](https://azure.microsoft.com/support/create-ticket).
```

Comment:
What are we trying to convey here:

"Calling these APIs with `az rest` is subject to change for reasons including, but not limited to,"

The **result** or **behavior** or **output** of `az rest` is subject to change?  If so, then how about ...

"The success of `az rest` is contingent upon permissions, the design of Microsoft Graph features, AAD Conditional Access policies, and other elements that impact REST API calls. For service REST API questions ..."

Reply:
The behavior of `az rest` itself is not likely to change very frequently. However, the **result** or **behavior** or **output** of the API that is being called is subject to change.

Reply:
@jiasli I think that we should provide more guidance on how to use properly `az rest` with some clear examples on how to perform a PUT, PATCH and GET operations on bash and PowerShell.

@dbradish-microsoft can we create a conceptual page on how to use az rest with a concrete example (I suggest container app, container registry for example).